test/league_stats_support_test.rb

```diff
@@ -0,0 +1,224 @@
+require_relative 'test_helper'
+require_relative '../lib/stat_tracker'
+require_relative '../lib/games'
+require_relative '../lib/teams'
+require_relative '../lib/game_teams'
+
+
+class LeagueStatsSupportTest < MiniTest::Test
+
+  def setup
+    locations = { games: './data/dummy_games.csv', teams: './data/teams.csv', game_teams: './data/dummy_game_teams.csv' }
+    @stat_tracker = StatTracker.from_csv(locations)
+  end
+
+  def test_generate_num_goals_per_team
+
+  example = {
+    "3" => 7,
+    "6" => 11,
+    "8" => 5,
+    "9" => 9,
+      "20" => 9,
```

Comment:
maybe make sure all indentation is consistent? Not sure if it just shows this way on Github

Reply:
Corrected, thank you ma'am